package.json

```diff
@@ -32,11 +32,11 @@
     "postswagger": "sed -i -r 's/([a-z1-9]+)_([a-z])/\\1\\U\\2/g' src/api/types.ts && yarn prettier"
   },
   "peerDependencies": {
-    "@patternfly/react-code-editor": "^4.1.25",
-    "@patternfly/react-core": "^4.97.2",
-    "@patternfly/react-icons": "^4.9.2",
-    "@patternfly/react-table": "^4.23.2",
-    "@patternfly/react-tokens": "^4.10.2",
+    "@patternfly/react-code-editor": "^4.2.92",
```

Comment:
We need to verify these changes in context of the uhc-portal project.

Reply:
+1

Reply:
Yes, it's exactly that. @jkilzi

Reply:
@jgyselov if you have the time we can make use of a screenshot in the PR's description
@mareklibra it turns out that PF version bumping is really necessary, I agree with you, we need to check we are good in `uhc-portal`

Reply:
@jgyselov please open a MR in uhc-portal similar [to this one](https://github.com/openshift-assisted/assisted-ui/pull/459), you can add the link to it here as well or in the JIRA task for better visibility